Modules/Core/Common/include/itkImageToImageFilter.h

```diff
@@ -99,7 +99,7 @@ namespace itk
  * \sphinxexample{Core/Common/MultipleInputsOfSameType,Multiple Inputs Of Same Type}
  * \sphinxexample{Core/Common/MultipleInputsOfDifferentType,Multiple Inputs Of Different Type}
  * \sphinxexample{Core/Common/MultipleOutputsOfSameType,Multiple Outputs Of Same Type}
- * \sphinxexample{Core/Common/MultThreadOilPainting,Mult-thread Oil Painting}
+ * \sphinxexample{Core/Common/MultiThreadOilPainting,Mult-thread Oil Painting}
```

Comment:
The spelling should be corrected in the human-readable string (Mult-thread Oil Painting).

Reply:
Thank you for catching this, should be resolved now.